.github/workflows/ci.yml

```diff
@@ -318,6 +318,56 @@ jobs:
       # 1.49.
       CARGO_PRIMARY_PACKAGE: 1
 
+  valgrind:
+    needs: [fmt]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - uses: Swatinem/rust-cache@v1
+        with:
+          key: cargo-valgrind
+        continue-on-error: true
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          profile: minimal
+      - uses: taiki-e/install-action@valgrind
+      - run: python -m pip install -U pip nox
+      # FIXME(adamreichold): Drop skip-macros when Valgrind understands DWARF5 as generated by LLVM 14,
+      # c.f. https://bugs.kde.org/show_bug.cgi?id=452758#c35
```

Comment:
Maybe you can try Rust 1.61.0 which still uses DWARF4.

Reply:
If that works, could be worth it given that's within out MSRV range.